src/containers/Root/UnauthorizedRoot.tsx

```diff
@@ -1,7 +1,22 @@
 import React from 'react';
 
+import { styled } from '../../styles';
+
+import { Login } from "../Login";
+
+const Container = styled.div`
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: ${({theme})=> theme.colors.backgroundGray};
+`;
+
 export const UnauthorizedRoot: React.FC = () => {
   return (
-    <></>
+    <Container>
```

Comment:
use Switch instead of Wrapper
use Login container inside Route component from 'react-router-dom'